modules/markdown.php

```diff
@@ -6,7 +6,7 @@
  * Sort Order: 31
  * First Introduced: 2.8
  * Requires Connection: No
- * Auto Activate: No
+ * Auto Activate: Yes
```

Comment:
If you're aiming to get this to happen on all upgrades, I think you'll want to move it outside of this conditional https://github.com/Automattic/jetpack/pull/6694/files#diff-9ba9fa5bb52bd6cf36c6c0bdd5ed4830R2793